Files/Dialogs/FilesystemOperationDialog.xaml.cs

```diff
@@ -23,6 +26,65 @@ public FilesystemOperationDialog(FilesystemOperationDialogViewModel viewModel)
             ViewModel = viewModel;
             ViewModel.View = this;
         }
+
+        private void MenuFlyoutItem_Click(object sender, Windows.UI.Xaml.RoutedEventArgs e)
+        {
```

Comment:
Why isn't it done in View Model 🤨 😄

Reply:
Because the code in the faction has to work directly with the ui, mainly ``DetailsGrid``.